lib/src/rules/email.ts

```diff
@@ -1,26 +1,15 @@
 import { FieldValidationResult, FieldValidationFunction } from '../entities';
+import { isValidPattern } from './pattern';
 
 // RegExp from http://emailregex.com
 const EMAIL_PATTERN = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
 
 export const email: FieldValidationFunction = (value: string) => {
   const validationResult = new FieldValidationResult();
-  const isValid = isValidEmail(value);
+  const isValid = isValidPattern(value, EMAIL_PATTERN);
 
   validationResult.succeeded = isValid;
   validationResult.type = 'EMAIL';
   validationResult.errorMessage = isValid ? '' : 'Please enter a valid email address.';
```

Comment:
The error message provided by the validationResult is just "courtesy", is not in the scope of the library or validators to handle multilanguage (the library would do too much). To handle multilangue there is another filed that "type" this field informs you the type of validation that has failed, there you can implement an ngMessage + Multilanguage approach (outside from the validator itself), similar approach as angular.

I think what @luisgonzalo is pointing is a good idea to open a new issue: implement multilanguage sample, @luisgonzalo would you like to be volunteer for implementing this case? we will guide you.